src/charm.py

```diff
@@ -1266,21 +1413,27 @@ def _initialise_replica_set(self, event: StartEvent) -> None:
                 )
                 event.defer()
                 self.status.set_and_share_status(
-                    WaitingStatus("waiting to initialise replica set")
+                    WaitingStatus("[ERR1] waiting to initialise replica set")
```

Comment:
I assume ERR1 + ERR2 were from local debugging? 

Let's remove them here. But also I can see why they are useful for debugging, perhaps we can have some more informative logs to help debug in future?

Reply:
Yes and it's a miss here, I indeed have forgotten to remove those lines which helped me understand what was going on here.